### PR TITLE
Register new package IntervalOptimisation.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaIntervals/IntervalOptimisation.jl](https://github.com/JuliaIntervals/IntervalOptimisation.jl)
Release: [v0.1.0](https://github.com/JuliaIntervals/IntervalOptimisation.jl/releases/tag/v0.1.0)
Travis: [Travis Build Status](https://travis-ci.org/JuliaIntervals/IntervalOptimisation.jl/builds/243857552)
cc: @dpsanders

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@dpsanders This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by ararslan** [comment]
Can we get an American English IntervalOptimi**z**ation.jl alias? :trollface:

**Comment by dpsanders** [comment]
Believe me, it was quite deliberate! ;)

File: IntervalOptimisation/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+
+IntervalArithmetic 0.9.1
+IntervalRootFinding 0.1.1
```
Comment (tkelman) [inline IntervalOptimisation/versions/0.1.0/requires]: does this rely on bugfixes that are only in the .1 patch tags?

**Comment by tkelman** [comment]
The travis failure at https://travis-ci.org/JuliaIntervals/IntervalOptimisation.jl/jobs/243859500 looks like there's a bug that needs fixing

File: IntervalOptimisation/versions/0.1.0/requires (hunk 1 above)
Comment (dpsanders) [inline IntervalOptimisation/versions/0.1.0/requires]: For IntervalArithmetic yes. I have changed the IntervalRootFinding lower bound down to 0.1.

**Comment by dpsanders** [comment]
Bug fixed and American English aliases added, thanks.

**Comment by dpsanders** [comment]
It's not clear that a package name alias is possible, but I'm happy for there to be one if it is.

**Comment by dpsanders** [comment]
Package tests are now passing.

**Comment by ararslan** [comment]
> Bug fixed and American English aliases added, thanks.

I was actually kidding, but cool!

**Comment by dpsanders** [comment]
Actually I'm not sure it was such a good idea, since it makes tab-completion pretty unfriendly.

**Comment by ararslan** [comment]
Yeah, in general it doesn't seem that great to have two subtly different ways to spell the same functions. Sorry, I should I have made it clearer that I didn't expect you to actually do that.

**Comment by dpsanders** [comment]
The idea was also raised here: https://github.com/JuliaIntervals/IntervalOptimisation.jl/issues/1

But I'm definitely having second thoughts.

**Comment by ararslan** [comment]
FWIW Hadley Wickham, the supreme deity of R, writes his R packages using British English (or New Zealand, I guess) but also provides aliases using American English spellings, so this isn't without precedent. I will also note though that the converse is extraordinarily rare: packages of any kind written in American English seem never to provide British English aliases, so it seems perhaps odd to expect BE -> AE.

**Pull request merged**